<launch>
	<node name="thermal_viewer" pkg="image_view" type="image_view" output="screen">
		<remap from="image" to="/thermal_camera/image_raw" />
		<param name="image_transport" value="raw" />
		<param name="autosize" value="false" />
	</node>
</launch>
